### DISCUSSION TITLE
Different Localications

### DISCUSSION THREAD

**Discussion by mickelsr** [hargata/lubelog#171] [Ideas]
Hey what i found pretty nice if you have the ability to change the language and currency. For Example German and EUR. :)

**Comment by hargata** [comment]
Currency localization is already in there, just have to change the LC_ALL and LANG variables in .env to de_DE

Language localization is being planned

**Comment by hargata** [comment]
@mickelsr the translation feature has been released, now just pending on German translators.
https://docs.lubelogger.com/Translations#creating-your-own-translation

Will now close this issue since the functionality to add translations has been released.
